pkgs/by-name/cl/clapgrep/package.nix

```
@@ -0,0 +1,74 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+
+  wrapGAppsHook4,
+  pkg-config,
+  blueprint-compiler,
+
+  gtk4,
+  libadwaita,
+  glib,
+
+  nix-update-script,
+}:
+let
+  version = "1.3.1";
+  appid = "de.leopoldluley.Clapgrep";
+in
+rustPlatform.buildRustPackage {
+  pname = "clapgrep";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "luleyleo";
+    repo = "clapgrep";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-MYV8MrCIpa8eqp2iCLTNLZrVQOyGsMEGqlnEF43fyls=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-tpWv8CznTxoAgOf0mb99peqUTQSfv+16dAmX8n7XVDQ=";
+
+  nativeBuildInputs = [
+    wrapGAppsHook4
+    pkg-config
+    blueprint-compiler
+  ];
+
+  buildInputs = [
+    gtk4
+    libadwaita
+    glib
+  ];
+
+  cargoFlags = [ "--package=clapgrep-gnome" ];
```

```suggestion
  cargoBuildFlags = [ "--package=clapgrep-gnome" ];
  cargoTestFlags = cargoBuildFlags;
```
I don't believe this is a recognized variable. Correct me if I'm wrong, though

---

It works without it so I'm just gonna remove it